package.json

```diff
@@ -4,21 +4,25 @@
   "private": true,
   "dependencies": {
     "axios": "0.19.1",
+    "brew": "0.0.8",
     "classnames": "2.2.6",
     "cookie-session": "^1.4.0",
     "cors": "2.8.5",
     "express": "^4.17.1",
     "express-session": "^1.17.1",
+    "install": "^0.13.0",
     "jsonwebtoken": "^8.5.1",
     "node-fetch": "^2.6.0",
+    "or": "^0.2.0",
     "pg": "8.0.3",
     "react": "16.10.2",
     "react-dom": "16.10.2",
     "react-modal": "3.11.2",
     "react-router-dom": "5.1.2",
     "react-scripts": "3.2.0",
     "url": "^0.11.0",
-    "virtualenv": "0.3.1"
+    "virtualenv": "0.3.1",
+    "yarn": "^1.22.5"
```

Comment:
It looks like these are your local changes from installation. Is that right? I don't think we need these changes for the purpose of this PR. If this is true, please remove changes from both `packages.json` and `yarn.lock` files.

Reply:
Hi Rey,

yes, we don't need packages.json and yarn lock changes, shall I create another request?
with only required changes?